code/modules/mob/living/carbon/human/species.dm

```diff
@@ -795,6 +795,11 @@
 
 			if(istype(J) && J.allow_thrust(0.01, H))
 				. -= 2
+			else
+				var/obj/item/organ/internal/cyberimp/chest/thrusters/T = H.getorganslot("thrusters")
+				if(istype(T) && T.allow_thrust(0.01, H))
+					. -= 2
+
```

Comment:
Hmm. the problem is that by moving it to carbon, you run into the issue of it stacking with the hardsuit or regular jetpacks.